docs/_layouts/ds6/default.html

```diff
@@ -1,17 +1,67 @@
 <!doctype html>
 <html lang="en">
     <head>
-        <title>eBay skin DS6</title>
+        <title>eBay Skin — for DS6</title>
         <meta name="viewport" content="width=device-width, initial-scale=1"/>
         <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
         <link type="text/css" rel="stylesheet" href="../static/ds6/skin.css"/>
         <link type="text/css" rel="stylesheet" href="../static/ds6/docs.css"/>
     </head>
     <body>
+        <header>
+            <h1>
+                <span class="inline-svg">
+                    <svg version="1.0" xmlns="http://www.w3.org/2000/svg" width="80pt" height="40pt" viewBox="0 0 182.000000 76.000000"
+                        preserveAspectRatio="xMidYMid meet" aria-hidden>
+                        <g transform="translate(0.000000,76.000000) scale(0.100000,-0.100000)" fill="#006efc" stroke="none">
+                            <path d="M450 614 l0 -137 -22 33 c-38 55 -97 80 -193 80 -69 0 -90 -4 -135
+                                -27 -75 -38 -100 -87 -100 -192 0 -124 51 -187 167 -211 105 -22 222 11 262
+                                73 l21 32 0 -47 0 -48 50 0 c43 0 50 3 50 19 0 18 3 18 47 -7 69 -38 169 -38
+                                235 1 26 15 55 40 64 56 l17 29 16 -31 c44 -88 191 -113 298 -51 42 24 43 24
+                                43 4 0 -17 6 -20 45 -20 l45 0 0 143 0 142 74 -140 74 -139 -39 -78 -39 -78
+                                57 0 58 0 138 269 c75 147 137 273 137 280 0 7 -19 11 -50 11 l-50 0 -76 -152
+                                -77 -153 -76 153 -76 152 -57 0 c-32 0 -58 -4 -58 -9 0 -6 -16 -3 -35 5 -21 8
+                                -67 14 -117 14 -100 0 -159 -23 -182 -69 -27 -51 -22 -61 30 -61 38 0 51 5 74
+                                30 23 25 36 30 74 30 25 0 56 -5 69 -11 23 -10 47 -46 47 -69 0 -6 -35 -10
+                                -79 -10 -99 0 -179 -15 -212 -39 l-26 -19 -12 54 c-15 64 -55 115 -112 145
+                                -53 27 -161 26 -214 -1 -22 -11 -41 -20 -42 -20 -2 0 -3 45 -3 100 l0 100 -55
+                                0 -55 0 0 -136z m-144 -113 c17 -11 36 -33 42 -50 l11 -31 -129 0 c-117 0
+                                -128 1 -123 18 19 72 127 107 199 63z m450 3 c65 -31 92 -127 57 -201 -22 -47
+                                -66 -73 -124 -73 -78 0 -129 57 -129 145 0 113 96 176 196 129z m504 -159 c0
+                                -9 -7 -31 -15 -51 -30 -72 -153 -96 -202 -39 -28 31 -28 37 -5 65 21 26 66 38
+                                155 39 54 1 67 -2 67 -14z m-810 -25 c0 -29 -1 -30 -51 -30 -42 0 -54 -4 -71
+                                -25 -39 -50 -135 -52 -187 -4 -21 20 -41 58 -41 80 0 5 73 9 175 9 l175 0 0
+                                -30z" />
+                        </g>
+                    </svg>
+                </span>
+                Skin <span class="secondary-text heading1__info">for DS6</span>
+            </h1>
+        </header>
+        <aside>
```

Comment:
Debatable whether an aside tag should be used here. Feels like this nav is closely related, rather than tangentially related, to the content of it's sectioning content (in this case the body tag, I believe).

Reply:
Easy to change. Let me know if you think I should just use the `<nav>`. Mostly this is a wrapper to keep the right border when the `<nav>` becomes fixed.